frontend/src/components/ResultsSection/ResultsSection.js

```diff
@@ -84,7 +84,7 @@ function constructData(){
       contextInfo.push(null)
     }
 
-    var domains = search_data.search_results[i]['lemma_wordform']['linguist_info']['rw_domains'] // please notes domain list can include "" item
+    var domains = search_data.search_results[i]['lemma_wordform']['rw_domains'].split(';').map((x) => x.trim()) // please notes domain list can include "" item
```

Comment:
The API should encode things better so you don't have to do this.  Eventually, we want to just get ['rw_domains'] and get a list of trimmed strings.  That's on my court though.